middleware/stations.js

```diff
@@ -4,24 +4,18 @@ function unenvelope(envelope) {
   return envelope['data']
 }
 
-function fetchStations(callback) {
-
-  fetch('http://pomiary.gdmel.pl/rest/stations')
+function fetchStations() {
+  return fetch('http://pomiary.gdmel.pl/rest/stations')
     .then((res) => res.json())
-    .then((json) => {
-      // console.log(json);
+    .then((json) => unenvelope(json))
```

Comment:
Funkcja unenvelope z tego co pamiętam zawacała tylko jeden poziom po kropce, nie ma co dodawać abstrakcji w takiej sytuacji według mnie

Reply:
+ jak masz tylko jeden argument, to możesz pominąć nawias, czyli json => unenvelope(json)

Reply:
Tru dat.

@olo2552 Co masz na myśli przez "dodawanie abstrakcji"? o.0